UNRELEASED.md

```diff
@@ -28,6 +28,9 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 
 - `bpk-component-infinity-gauntlet`:
   - New `timeStone` prop for controlling time. See &lt;link to docs site&gt;.
+- `bpk-component-slider`:
```

Comment:
We usually add the entries to another place in this file – please check out the history for samples :)

Reply:
Addressed by 70f79058d0e1eb33ed92033bb0630c6939103cc2.